server/routes/comments.js

```diff
@@ -34,8 +36,106 @@ const middleWare = async (req, res, next) => {
 }
 
 
+
+const getAllCommentsFromASinglePost =  async(req, res) => {
+  let post_id = req.params.post_id
+  try {
+    let response = await db.any(`SELECT * FROM comments WHERE post_id = ${post_id}`);
+    // let response = await db.any(`SELECT body FROM comments WHERE post_id = ${post_id}`);
+    res.json({
+        status: "success",
+        message: `got all the comments from ${post_id}`,
+        body: response
+    });
+  } catch (error) {
+      res.status(500)
+      res.json({
+          message: "Error. Something went wrong"
+      })
```

Comment:
Not much of a major change, but by console.log-ing the error, we can get a few more clues from the terminal on what to fix.